<?xml version="1.0"?>
<launch>

  <node pkg="controller" type="controller_node" respawn="false" name="controller" output="screen">
    <rosparam file="$(find controller)/config/controller_config.yaml" command="load" />
  </node>

  <node name="rviz_controller" pkg="rviz" type="rviz" args="-d $(find controller)/src/ros_viewer/controller.rviz"/>
  
</launch>